.github/workflows/transport-amqp.yml

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+name: transport-amqp
+on:
+  push:
+    paths:
+      - 'component/didcomm/transport/amqp/**'
```

Comment:
You need also to ignore this path for a common workflow.
See https://github.com/hyperledger/aries-framework-go-ext/blob/main/.github/workflows/common.yml#L10 and https://github.com/hyperledger/aries-framework-go-ext/blob/main/.github/workflows/common.yml#L15

Reply:
Yeah, I noticed that pattern when doing the mongo driver.  I'll fix these.